Pcap++/header/DeviceUtils.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
```

Comment:
suggest to put all util functions in one file, as there is not much code.

Reply:
I see. originally I thought we could merge `MemoryUtils` and `DeviceUtils`, but it did make sense to be separated.